server/appServices.js

```diff
@@ -92,13 +115,15 @@ function getServicesForApp(namespace, app, kubeclient) {
 
 async function updateAppsAndWatch(namespace, kubeclient) {
   connectionAttempts++;
-  updateAll(namespace, kubeclient).then(() => {
+  logAction('About to call updateAll()');
+  updateAll(namespace, kubeclient).then(pickles => {
```

Comment:
Why that `pickles` parameter?

Reply:
It was my random naming when trying to figure out what was happening. I will refactor